generator/tools/templates/react/src/hooks/useEffectOnce.tsx

```diff
@@ -0,0 +1,42 @@
+import { useRef, EffectCallback, DependencyList, useEffect } from "react";
```

Comment:
We should rename this file along with whatever we agree useEffectOnce/useEffectHmr should be called

Reply:
Correct, I'm on it.